pkgs/development/compilers/dotnet/sdk/allow-missing-pdbs.patch

```diff
@@ -0,0 +1,25 @@
+From 6428e996472d2334f71da5e54f96f3a19f992f5e Mon Sep 17 00:00:00 2001
+From: David McFarland <corngood@gmail.com>
+Date: Fri, 29 Dec 2023 23:24:25 -0400
+Subject: [PATCH 1/2] allow missing pdbs
+
+---
+ build.proj | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/build.proj b/build.proj
+index 7d074d8802..0fdfd99696 100644
+--- a/build.proj
++++ b/build.proj
+@@ -123,7 +123,7 @@
+     <CreateSdkSymbolsLayout SdkLayoutPath="$(SdkLayout)"
+                             AllSymbolsPath="$(UnifiedSymbolsLayout)"
+                             SdkSymbolsLayoutPath="$(SdkSymbolsLayout)"
+-                            FailOnMissingPDBs="true" />
++                            FailOnMissingPDBs="false" />
```

Comment:
Are you seeing the pdb failure on specific architectures/environments or everywhere?

Reply:
I'm currently only using this on darwin, and only in stage0:

```
    ++ lib.optionals stdenv.isDarwin [
      ./sign-apphost.patch
      # /private/tmp/nix-build-dotnet-vmr-8.0.2.drv-0/dotnet-8.0.2/build.proj(123,5): error : Did not find PDBs for the following SDK files:
      # /private/tmp/nix-build-dotnet-vmr-8.0.2.drv-0/dotnet-8.0.2/build.proj(123,5): error : sdk/8.0.102/System.Resources.Extensions.dll
      # /private/tmp/nix-build-dotnet-vmr-8.0.2.drv-0/dotnet-8.0.2/build.proj(123,5): error : sdk/8.0.102/System.CodeDom.dll
      # /private/tmp/nix-build-dotnet-vmr-8.0.2.drv-0/dotnet-8.0.2/build.proj(123,5): error : sdk/8.0.102/FSharp/System.Resources.Extensions.dll
      # /private/tmp/nix-build-dotnet-vmr-8.0.2.drv-0/dotnet-8.0.2/build.proj(123,5): error : sdk/8.0.102/FSharp/System.CodeDom.dll
      ./allow-missing-pdbs.patch
    ];
```

I'll run a few more tests in case it's nondeterministic or something.

Reply:
Thanks. That's very interesting. I see this issue only on some environments that are running linux + s390x.

Reply:
I reported to .NET folks: https://github.com/dotnet/source-build/issues/4150